src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs

```diff
@@ -9,6 +10,14 @@ namespace AndcultureCode.CSharp.Sitefinity.Core.Interfaces
 {
     public interface IDynamicContentRepositoryConductor
     {
+        /// <summary>
+        /// Creates a Dynamic Content of the specified type T.
```

Comment:
Suggest reworking to something like `Creates dynamic content of the specified type`.  The proper casing around `Dynamic Content` implies makes it appear to be an object which I think should be more generically stated in the description.